dashboard/db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20180425175740) do
+ActiveRecord::Schema.define(version: 20180428013942) do
```

Comment:
Not sure how this snuck in, it's not relevant to this PR, I'll remove before merging.